bots/code-analysis-bot.js

```diff
@@ -321,6 +347,9 @@ process.stdin.on('end', function() {
   const owner = process.env.GITHUB_OWNER;
   const repo = process.env.GITHUB_REPO;
 
+  process.env.GITHUB_PR_NUMBER = 30444;
+  process.env.GITHUB_TOKEN = '78a72af35445ca3f8180b1a98e0bbd56ff1ccba1';
```

Comment:
This token was disabled by GitHub today once this commit was added to your fork.

When including a GitHub token in a change, always make sure to split it in two. Otherwise, GitHub's OAuth token invalidation service will kick in and disable the token, which we don't want to happen here (we're intentionally making this token public).

Reply:
Ah, sorry this wasn't supposed to be committed at all. Is there a way to create a new token? Presumably this is something I cant do myself?